src/Adapter/AdapterAbstractServiceFactory.php

```diff
@@ -49,35 +48,35 @@ public function canCreateServiceWithName(ServiceLocatorInterface $services, $nam
     /**
      * Create a DB adapter
      *
-     * @param  ServiceLocatorInterface $services
-     * @param  string $name
+     * @param  ContainerInterface $container
      * @param  string $requestedName
+     * @param  array $options
      * @return Adapter
      */
-    public function createServiceWithName(ServiceLocatorInterface $services, $name, $requestedName)
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
-        $config = $this->getConfig($services);
+        $config = $this->getConfig($container);
         return new Adapter($config[$requestedName]);
     }
 
     /**
      * Get db configuration, if any
      *
-     * @param  ServiceLocatorInterface $services
+     * @param  ContainerInterface $container
      * @return array
      */
-    protected function getConfig(ServiceLocatorInterface $services)
+    protected function getConfig(ContainerInterface $container)
     {
         if ($this->config !== null) {
             return $this->config;
         }
 
-        if (!$services->has('Config')) {
+        if (!$container->has('Config')) {
```

Comment:
zend-mvc now normalizes this service to `'config'` (all lowercase). Since the SM does not normalize, we need all these references updated as well.